### ISSUE TITLE
Federative mode

### ISSUE THREAD

**Issue by nickkolok** [IPTnet/ipt_connect#201]
As far as `ipt_connect` can be as decentralized as the tournament movement, we can imagine smth like federative mode, when some servers mirror information from others. Users and signatures, public and private keys, blockchain and p2p - fell free to share your ideas :)

It will be also important to save the data about past tournaments and to perform data analysis.  Mirroring a tournament (with almost no recalculations) looks like a rather simple work for a server.

See also #200

**Comment by david2457** [comment]
Fantastic idea!
If this works this would be excellent!

**Comment by lysenko-nikolay** [comment]
Yep, i have thought about combining and merging all IPT-connect branches from 2016.

This would be great not only to assess performance level of the participants, but also investigate whether specific jury member is adequate or not 😄 

But i think that making such an unification between IPT editions will be tough.
